aleph/settings.py

```diff
@@ -64,6 +64,10 @@
 # Assumes a comma-separated list.
 ADMINS = env.to_list("ALEPH_ADMINS")
 
+def is_admin(email):
```

Comment:
I think we should move this to `logic/roles.py` and rename it to something like `is_auto_admin` to be more explicit. We try not to put app logic in settings directly.

Reply:
It's a bit tricky — this causes circular dependency in imports: `aleph.core → aleph.oauth → core.logic.roles → aleph.core`

I think the right solution would be splitting `aleph.core` into core that would provide things like `db` and `cache` that would be imported in different components, and `aleph.app` that will only import everything needed to start the app.

For now moved `is_auto_admin` to `aleph.util`